### PR TITLE
Accessibilitycheck

### PR DESCRIPTION
**Pull request by rubenvolman**
First adjustments to test changes in external report tool

### CODE REVIEW

**Comment by rubenvolman** [comment]
#26 pull request adjustments are included in this one.

File: public/index.html [hunk 1]
```diff
@@ -160,7 +160,7 @@ <h3>Why use POAP?</h3>
         <div class="container">
           <div class="container-faqs">
             <div class="faq faqs-third-text" data-aos="fade-right">
-              <h3>How does it work?</h3>
```
[the remark this answers is not in the document]
Comment (rubenvolman) [reply public/index.html:163]: :white_check_mark: Re-checked font settings

File: public/index.html [hunk 2]
```diff
@@ -82,7 +82,7 @@
       <div class="banner-container">
         <div class="banner-content">
           <h1 id="start" class="start-box"><span class="description-title">POAP - The bookmarks of your life</span></h1>
-          <p class="subtitle">POAPs are digital mementos, minted in celebration of life's remarkable moments.</p>
+          <h2 class="subtitle">POAPs are digital mementos, minted in celebration of life's remarkable moments.</h2>
```
[the remark this answers is not in the document]
Comment (rubenvolman) [reply public/index.html]: :white_check_mark: Resolved and changed back to old style

File: public/assets/scss/globals/_boilerplate.scss [hunk 3]
```diff
@@ -134,12 +134,12 @@ h1 {
 h2 {
     @include x-rem(font-size, 40px);
     color:$color-primary;
-    font-family: $font-family-narrow;
+    font-family: 'Comfortaa';
```
[the remark this answers is not in the document]
Comment (rubenvolman) [reply public/assets/scss/globals/_boilerplate.scss]: :white_check_mark: Variables done

File: public/index.html [hunk 4]
```diff
@@ -232,7 +235,7 @@ <h3>How can I contact you or get involved?</h3>
   <!-- FOOTER -->
   <footer class="footer footer-grid">
     <!-- FIX: this POAP image generates a js error when hovered -->
-    <a href="/" class="home-link"><img src="assets/images/POAP.svg" alt="" /></a>
+    <a href="/" class="home-link"><img src="assets/images/POAP.svg" alt="POAP" /></a>
```
[the remark this answers is not in the document]
Comment (rubenvolman) [reply public/index.html]: :wheelchair: Made some starter adjustments. :white_check_mark:

File: public/assets/scss/globals/_boilerplate.scss [hunk 5]
```diff
@@ -1,5 +1,5 @@
 @import "fonts";
-
+@import "../../../../node_modules/bootstrap/scss/vendor/rfs";
```
Comment (actuallymentor) [inline public/assets/scss/globals/_boilerplate.scss:2]: Why this import? Bootstrap already imports this with `import "bootstrap"` right?

**Review by actuallymentor — COMMENTED** [review]
See the one import comment. If that has a good reason you are good for merge.

File: public/assets/scss/globals/_boilerplate.scss (hunk 5 above)
Comment (rubenvolman) [reply public/assets/scss/globals/_boilerplate.scss:2]: This import gives dev access to the Bootstrap specific rfs mixin `font-size`. Although rfs is included in the build its mixins are not is not in the dev environment.

**Pull request merged**